src/lib/utils.tsx

```diff
@@ -85,15 +88,18 @@ export function sum(arr: Array<number>): number {
 }
 
 export function urlencode(obj: any): string {
-  return btoa(String.fromCharCode.apply(null, Msgpack.encode(obj)));
+  return btoa(
+    // @ts-ignore
+    String.fromCharCode.apply(null, Msgpack.encode(obj) as Uint8Array),
+  );
 }
 
 export function urldecode(obj: string): any {
   return Msgpack.decode(
     new Uint8Array(
       atob(obj)
         .split("")
-        .map((c) => c.charCodeAt(0))
-    )
+        .map((c) => c.charCodeAt(0)),
+    ),
```

Comment:
**thought:** we should actually put prettier in the project, #33

Reply:
I've now opened #85; we could merge that first and then rebase to make the diff here more manageable.

Reply:
rebased with main @psvenk